frameworks/JavaScript/uwebsockets.js/src/utils.js

```diff
@@ -57,3 +59,33 @@ export function escape(text) {
   if (unsafeHTMLMatcher.test(text) === false) return text;
   return text.replace(unsafeHTMLMatcher, function (m) { return escapeHTMLRules[m] || m; });
 }
+
+/**
+ * Using Slow json stringify module to get faster results as explained in just-js blog post below
```

Comment:
Thank you, but sorry if I wasn't clear enough. What I mean is, could you remove any references in the code comments that are not directly related to uWebsockets.js? Instead, you can put the references in the PR conversation section.